package.json

```
@@ -4,13 +4,8 @@
   "main": "modules/react-native-status/nodejs/bindings.js",
   "private": true,
   "scripts": {
-    "start": "react-native start",
-    "ios": "react-native run-ios",
-    "preinstall": "make status-go-library",
-    "app:compile:android": "shadow-cljs compile android",
-    "app:watch": "shadow-cljs watch android",
-    "app:packager": "react-native start --host 0.0.0.0 --port 8081",
-    "app:android": "react-native run-android",
+    "preinstall": "scripts/check-nix-shell.sh && make status-go-library",
```

Since the purpose of this is to prevent `yarn install` from running outside of Nix shell, and not just `preinstall` maybe we should add a "comment" to that effect:
```suggestion
    "_comment" "This also prevents 'yarn install' from running outside of nix.",
    "preinstall": "scripts/check-nix-shell.sh && make status-go-library",
```